### PR TITLE
Add information that we need forced failover on AG side after Forced …

### PR DESCRIPTION
**Pull request by MSFTPawelM**
…Quorum was used

This article is not clear enough, we need to clearly state that after we used forced quorum, forced failover is necessary on AG side to bring the group back to usable state -  as described in the https://docs.microsoft.com/en-us/sql/database-engine/availability-groups/windows/perform-a-forced-manual-failover-of-an-availability-group-sql-server?view=sql-server-ver16&source=docs article:
After forcing quorum on the WSFC cluster (forced quorum), you need to force failover each availability group (with possible data loss). Forcing failover is required because the real state of the WSFC cluster values might have been lost. However, you can avoid data loss, if are able to force failover on the server instance that was hosting the replica that was the primary replica before you forced quorum or to a secondary replica that was synchronized before you forced quorum.

### CODE REVIEW

**Comment by Court72** [comment]
@MashaMSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ ms.author: mathoma
 5.  **Recover availability group database replicas as needed.** Non-availability group databases should recover and come back online on their own as part of the regular SQL Server startup process.  
   
      You can minimize potential data loss and recovery time for the availability group replicas by bringing them back online in this sequence:  primary replica, synchronous secondary replicas, asynchronous secondary replicas.  
+     
+>[!NOTE]
```
Comment (MashaMSFT) [inline docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md:88]:
```suggestion
   >[!NOTE]
```

**Comment by PRMerger3** [comment]
@MashaMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md [hunk 2]
```diff
@@ -84,6 +84,9 @@ ms.author: mathoma
...
+>After using forced quorum it is necessary to perform a forced failover with possible data loss to bring the availability group back online. For more information, review [Perform a forced manual failover of an availability group &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/perform-a-forced-manual-failover-of-an-availability-group-sql-server.md).
```
Comment (MashaMSFT) [inline docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md:89]:
```suggestion
   >After using forced quorum it is necessary to perform a forced failover with possible data loss to bring the availability group back online. For more information, review [Perform a forced manual failover of an availability group &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/perform-a-forced-manual-failover-of-an-availability-group-sql-server.md).
```

**Review by MashaMSFT — CHANGES_REQUESTED** [review]

**Comment by PRMerger15** [comment]
@MashaMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MashaMSFT** [comment]
#sign-off

File: docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md [hunk 3]
```diff
@@ -84,6 +84,9 @@ ms.author: mathoma
 5.  **Recover availability group database replicas as needed.** Non-availability group databases should recover and come back online on their own as part of the regular SQL Server startup process.  
   
      You can minimize potential data loss and recovery time for the availability group replicas by bringing them back online in this sequence:  primary replica, synchronous secondary replicas, asynchronous secondary replicas.  
+     
+   >[!NOTE]
```
Comment (MashaMSFT) [inline docs/sql-server/failover-clusters/windows/wsfc-disaster-recovery-through-forced-quorum-sql-server.md:88]:
```suggestion
   > [!NOTE]
```

**Comment by PRMerger5** [comment]
@MashaMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MashaMSFT** [comment]
#sign-off

**Pull request merged**